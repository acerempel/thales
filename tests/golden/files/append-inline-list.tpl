{for item in append(["potato","leek"],["squash","turnip","bobcat"])}
{item}{end}
