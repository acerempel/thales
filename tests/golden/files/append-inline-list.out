
potato
leek
squash
turnip
bobcat
